src/dcmstack/extract.py

```diff
@@ -307,8 +307,8 @@ def get_text(byte_str):
                        'OW or OB' : get_text,
                        'OB or OW' : get_text,
                        'UN' : get_text,
-                       'PN' : unicode,
-                       'UI' : unicode,
+                       'PN' : str,
```

Comment:
This is a change of behavior on Python 2 - I don't know whether that matters.  @moloney ?  If it does, then you can do a conditional import at the top of the file something like:

```
unicode_str = unicode if sys.version_info[0] < 3 else str
```